src/components/trending-developers/TrendingDevelopers.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import { getTrendingDevelopers } from '../../services/github';
+import styles from './TrendingDevelopers.module.css';
+
+function TrendingDevelopers() {
+  const history = useHistory();
+  const [trendingDevelopers, setTrendingDevelopers] = useState();
+
+  useEffect(() => {
+    async function fetch() {
+      try {
+        const result = await getTrendingDevelopers();
+        setTrendingDevelopers(result);
+      } catch (err) {
+        // Fail silently
```

Comment:
What about handling the error with a state?

Reply:
Yeah, I thought about this, but the component renders a list, so I'm uncertain if rendering an error to the user is the right UI pattern; that's why I merged the "can't fetch data" and "trending developers is empty" errors:

https://github.com/gabrieledarrigo/github-cards/pull/3/files#diff-94f6d0873bc97d5fe79aa64a8505e5a9R54

Don't know, have you got better ideas?

Reply:
Uhm, that's also true. We can leave it like this for now.

Reply:
Ok, you are welcome!